### DISCUSSION TITLE
问下LogHandler如何使用？文中例子看不出来

### DISCUSSION THREAD

**Discussion by davidwang863** [xxl-job/xxl-job-executor-go#31] [Q&A]
示例中只有这么一个例子，日志的存储和查询都是要自己实现吗？

exec.LogHandler(func(req *xxl.LogReq) *xxl.LogRes {
		return &xxl.LogRes{Code: 200, Msg: "", Content: xxl.LogResContent{
			FromLineNum: req.FromLineNum,
			ToLineNum:   2,
			LogContent:  "这个是自定义日志handler",
			IsEnd:       true,
		}}
	})

**Comment by rushuinet** [comment]
日志的存储和查询都是要自己实现的，这里提供了接口和一个接口实现示例，对接自己的日志系统即可
